api/v1alpha1/limitador_types.go

```diff
@@ -324,6 +327,10 @@ type Ports struct {
 	GRPC int32 `json:"grpc,omitempty"`
 }
 
+type Tracing struct {
+	Endpoint *string `json:"endpoint,omitempty"`
```

Comment:
Should `endpoint` be optional or required?

`spec.tracing` is optional, but `spec.tracing.endpoint`?

Reply:
Yeah that's a good point actually, if the tracing struct is provider the endpoint should be required I guess